pkg/datapath/loader/template.go

```diff
@@ -282,6 +282,6 @@ func elfVariableSubstitutions(ep datapath.Endpoint) map[string]uint64 {
 // ELFSubstitutions fetches the set of variable and map substitutions that
 // must be implemented against an ELF template to configure the datapath for
 // the specified endpoint.
-func ELFSubstitutions(ep datapath.Endpoint) (map[string]uint64, map[string]string) {
+func (l *loader) ELFSubstitutions(ep datapath.Endpoint) (map[string]uint64, map[string]string) {
```

Comment:
Nit: can this and its dependent functions be moved to package endpoint?

Reply:
Actually, its only used internally, except for in the tests of `pkg/datapath/linux/config`. And I don't see a what to not brick the tests. We should merge the header file writing logic of `pkg/datapath/linux/config` into `pkg/datapath/loader` at which point we can make it an internal function